canary/generator/download/query.py

```diff
@@ -0,0 +1,17 @@
+def build_query_string(dates, metric, channel, version, os=None, os_version=None,
```

Comment:
Can we add pytest test coverage for, at least, the core functions like this one? This way we could enable automated test coverage (in the future) and have some confidence that we don't break the code when making changes.

Reply:
You can add basic test coverage in this PR, no CI integration. We'll add more test coverage and enable CI in a future PR.